core-dev/packages/coqide-server/coqide-server.dev/url

```diff
@@ -0,0 +1 @@
+src: "git+https://github.com/coq/coq.git#master"
```

Comment:
@mattam82 please use `url` clauses in the `opam` file instead of putting the URL in a different file.

Reply:
If you wanna run the test suite, you need to depend on `coq-stdlib` too.

Reply:
I thought it was better not to put the url in the file so that the opam file can be exactly the same as in the git repo?

Reply:
This was discussed before, but the conclusion by Enrico (I believe) is that the separate-url-file approach is obsolete and likely to be removed in future opam versions.